commands/shop.py

```diff
@@ -48,46 +48,56 @@ async def shop(self, ctx):
                         add_balance(ctx.author, -15.00)
                         await ctx.send("Purchase Successful, Please contact a member of staff to request your tag!")
                     else:
-                        await ctx.send("Insufficient Funds")
+                        em = discord.Embed(title = "Purchase Failed", description = "Insufficient Funds", color = colour.reds)

```

Comment:
We use author fields to set embed titles, looks nicer in Discord

Reply:
An Example
```
embed = discord.Embed(description=coinflip_string, color=colour.primary)
embed.set_author(name="Coinflip")
```

Reply:
Also we don't wrap the `=` in spaces

Reply:
Working on the `=` now

Reply:
Fixed in b3bf5e3